source/extensions/transport_sockets/tls/utility.cc

```diff
@@ -71,21 +71,49 @@ Envoy::Ssl::CertificateDetailsPtr Utility::certificateDetails(X509* cert, const
   return certificate_details;
 }
 
+bool Utility::labelWildcardMatch(absl::string_view dns_label, absl::string_view pattern) {
+  if (pattern.empty()) {
```

Comment:
This can't be empty: the caller found `*` in the string

Reply:
ack. removed this check. The unit tests also didn't reach this line